crates/ef-testing/src/models/result.rs

```diff
@@ -2,35 +2,40 @@ use blockifier::{
     execution::call_info::CallInfo,
     transaction::objects::{TransactionExecutionInfo, TransactionExecutionResult},
 };
-use starknet::{core::types::FieldElement, macros::selector};
-use starknet_api::{hash::StarkFelt, transaction::EventContent};
+use starknet::macros::selector;
+use starknet_api::transaction::EventContent;
 use tracing::{error, info, warn};
 
 pub(crate) fn log_execution_result(
     result: TransactionExecutionResult<TransactionExecutionInfo>,
     case_name: &str,
     parent_dir: &str,
 ) {
+    let version = get_kakarot_version();
+
     match result {
         TransactionExecutionResult::Ok(info) => {
             if let Some(err) = info.revert_error {
                 warn!(
-                    "{}::{} reverted:\n{}",
+                    "{}::{} version {version} reverted:\n{}",
                     parent_dir,
                     case_name,
                     err.replace("\\n", "\n")
                 );
             } else {
                 info!(
-                    "{}::{} passed: {:?}",
+                    "{}::{} version {version} passed: {:?}",
                     parent_dir, case_name, info.actual_resources
                 );
+                #[cfg(feature = "v0")]
                 if let Some(call) = info.execute_call_info {
+                    use starknet::core::types::FieldElement;
+                    use starknet_api::hash::StarkFelt;
                     let events = get_kakarot_execution_events(&call);
                     // Check only one execution event.
                     if events.len() != 1 {
                         warn!(
-                            "{}::{} failed to find the single execution event: {:?}",
+                            "{}::{} version v0 failed to find the single execution event: {:?}",
```

Comment:
you can use `{version}` so that when it's updated in v1 there is no need to update this. And to keep the version name at one single place